data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -603,6 +616,21 @@
             speaker=Kaleh
             message= _ "How can he move that fast? He is faster than any goblin rider I have ever seen. It’s almost unnatural."
         [/message]
+
+        [if]
+            [variable]
+                name=sneak_up
+                boolean_equals=yes
+            [/variable]
+            [then]
+                [message]
+                    speaker=Nym
+                    message= _ "Spotted immediately! Well, I guess I spoke too soon about trying sneaking."
```

Comment:
Seems like this should be in `# Event 1: goblins appear in nearby foothills` rather than `# Event 3`. I'd put it where Zhul's "Goblins are so predictable" is, and hide Zhul's line if this triggers.

If it goes in event 3, it should go before Kaleh's line. Kaleh changes the subject to be about the rider's abilities.

Reply:
I put it in event 1 because from playing the scenario, **event 3 always occurs first no matter what**, and the line given by the enemy unit **actually exclaims that he has spotted elves**.

The timing is more natural this way.

About making the line be before Kaleh's, sure i can do that.

Reply:
That seems like a bug - without the map, event 1 triggers first (probably on turn 2). With the map, event 3 triggers when Panok captures a village on turn 1.

Panok's comment makes little sense on turn 1, because Panok says the elves are stronger than they thought. At this point, he's only seen 1 castle full of elvish recruits/recalls.

Reply:
Well, it's okay I can find a way to make the dialog make sense for occurring at Event 1.